generator/Cargo.toml

```diff
@@ -5,13 +5,15 @@ authors = ["Maik Klein <maikklein@googlemail.com>"]
 edition = "2018"
 
 [dependencies]
-vk-parse = { version = "0.5.0", features = ["vkxml-convert"] }
-vkxml = "0.3"
-nom = "6.0"
 heck = "0.3"
-proc-macro2 = "1.0"
 itertools = "0.10"
+nom = "6.0"
+once_cell = "1.7"
+proc-macro2 = "1.0"
 quote = "1.0"
+regex = "1.4"
```

Comment:
I am sort of ok with this in the generator. Just a bit odd because we also have nom.

Reply:
I don't like the regex itself, there should be a simpler way to consume some trailing digits and insert an `_` before there (if it wasn't already). At least thanks to #426 any errors caused by it will be caught.

`nom` is more useful for syntax parsing, but yes, after experimenting with it in https://github.com/MaikKlein/ash/tree/autgenerate-define-exprs we could use it to split out type names in a name and optional trailing version. Perhaps we can write a cleaner parser that takes care of separating out the `vk` prefix, type name, version, "type" (bits/flags), and vendor.